### ISSUE TITLE
Cada vez que você faz um force unwrap, um gatinho morre

### ISSUE THREAD

**Issue by lfarah** [iOSDevBR/iosdevbr-hubot#22]
A piada começou com a [Erica Sadun](https://twitter.com/ericasadun/status/638502420696305664) e acredito que tem muito fundamento.
Podemos analisar os text snippets e ver se alguém fez um force unwrap

**Comment by DougFischer** [comment]
Consegue sugerir o código para o reconhecimento?

**Comment by lfarah** [comment]
@DougFischer os caras do [SwiftLint](https://github.com/realm/SwiftLint) já tem [um algoritmo pra reconhecer force unwrap](https://github.com/realm/SwiftLint/blob/990f1d105c637cefded58a62c53956f6f378b039/Source/SwiftLintFramework/Rules/ForceUnwrappingRule.swift)

**Comment by marcelofabri** [comment]
Lembrando que o SwiftLint usa a AST do Swift e mesmo assim há alguns casos de falso positivo com essa regra 😅

**Comment by lfarah** [comment]
@marcelofabri verdade. [Eles sempre estão tentando melhorar esses falsos positivos](https://github.com/realm/SwiftLint/commit/b30a63bde31d2a3109068c4f519ae93ef73dd05e)
